data/frequencies/grab_wortschatz_data.py

```diff
@@ -12,12 +13,13 @@
 import requests
 
 
-WORTSCHATZ_DICT_PATH = "wortschatz_languages.json"
+_THIS_DIR = os.path.dirname(__file__)
+WORTSCHATZ_DICT_PATH = os.path.join(_THIS_DIR, "wortschatz_languages.json")
 
 
 def download(data_to_grab: Dict[str, Any]) -> Dict[str, Any]:
     to_retry = {}
-    os.mkdir("tgz")
+    os.makedirs("tgz", exist_ok=True)
```

Comment:
I had reason to not do it this way: I kept getting partial downloads and I couldn't figure out what was going on except via starting over. Just mentioning...

Reply:
I might suggest instead that on running the script it should annhilate the dirs.

Reply:
In my workflow this time, I found it convenient to have `os.makedirs(..., exist_ok=True)`, since I used the same dirs consistently, and I just let the scripts overwrite the files inside these dirs as I iterated. Perhaps whoever prepares the shared task data next year may find another approach more convenient.